doc/classes/Line2D.xml

```diff
@@ -16,8 +16,11 @@
 			</return>
 			<argument index="0" name="position" type="Vector2">
 			</argument>
+			<argument index="1" name="at_position" type="int" default="-1">
```

Comment:
It's confusing to have arguments named `position` and `at_position` when the two positions have nothing to do with each other. Maybe use `at_index`?

Reply:
Maybe is `offset` will be better

Reply:
I agree that it could be misleading. Curve3D and Curve2D also use `position` and `at_position` in their `add_point` method, so should it be renamed there too ?

Reply:
Yeah it would probably be good to rename it everywhere. I merged this PR nevertheless as it can be done in a new PR for all three classes.